SessionMessagingKit/Utilities/ObservableKey+SessionMessagingKit.swift

```diff
@@ -47,6 +47,7 @@ public extension ObservableKey {
     // MARK: - Conversations
     
     static let conversationCreated: ObservableKey = "conversationCreated"
+    static let conversationPinnedPriorityChanged: ObservableKey = "conversationPinnedPriorityChanged"
```

Comment:
To be consistent with the other event naming it might be good to rename this to `anyConversationPinnedPriorityChanged`